pkg/deploy/api/types.go

```diff
@@ -314,14 +314,23 @@ type DeploymentConfigSpec struct {
 
 // DeploymentConfigStatus represents the current deployment state.
 type DeploymentConfigStatus struct {
-	// LatestVersion is used to determine whether the current deployment associated with a DeploymentConfig
-	// is out of sync.
+	// LatestVersion is used to determine whether the current deployment associated with a deployment
+	// cofnig is out of sync.
 	LatestVersion int
-	// Details are the reasons for the update to this deployment config.
-	// This could be based on a change made by the user or caused by an automatic trigger
+	// Details are the reasons for the update to this deployment config. This could be based on a
+	// change made by the user or caused by an automatic trigger.
 	Details *DeploymentDetails
-	// ObservedGeneration is the most recent generation observed by the controller.
+	// ObservedGeneration is the most recent generation observed by the deploymentconfig controller.
```

Comment:
Also I still think if we're not specially calling it a `DeploymentConfig` proper noun, since this is user facing doc, we should use "configuration" over "config".